frontend/src/metabase/entities/actions/actions.ts

```diff
@@ -71,6 +76,29 @@ const associateAction = ({
     requires_pk: false,
   });
 
+const defaultImplicitActionCreateOptions = {
+  insert: true,
+  update: true,
+  delete: true,
+};
+
+const enableImplicitActionsForModel =
+  async (modelId: number, options = defaultImplicitActionCreateOptions) =>
+  async (dispatch: Dispatch) => {
+    await Promise.all(
+      Object.entries(options)
+        .filter(([, shouldCreate]) => !!shouldCreate)
+        .map(([method]) =>
```

Comment:
I personally find this a bit hard to follow, had to pause at this block for a minute to parse what exactly is happening here

Reply:
very fair. took me a second to figure out what this did too. I've cleared it up a bit.